openrouteservice/pom.xml

```diff
@@ -403,6 +403,17 @@
         <version>2.9.2</version>
         <scope>compile</scope>
     </dependency>
+    <dependency>
```

Comment:
Is there a reason why we need these? I just removed them from the pom and everything seemed to build/run ok

Reply:
When I remove it, the application fails to build